server/src/main/java/com/paypal/selion/proxy/SeLionSauceProxy.java

```diff
@@ -47,7 +47,7 @@
 
     private static final SeLionGridLogger LOGGER = SeLionGridLogger.getLogger(SeLionSauceProxy.class);
 
-    private int maxTestCase = 0;
+    private int maxTestCase;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Private field 'maxTestCase' could be made final; it is only initialized in the declaration or constructor.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1271346908/issues/source?bid=2928425&fileBranchId=2942499#l50)